src/NServiceBus.Core/Routing/DistributionStrategy.cs

```diff
@@ -24,6 +27,14 @@ protected DistributionStrategy(string endpoint, DistributionStrategyScope scope)
         public abstract string SelectReceiver(string[] receiverAddresses);
 
         /// <summary>
+        /// Selects a destination instance for a message from all known addresses of a logical endpoint.
```

Comment:
I try to tick to https://github.com/Particular/NServiceBus/blob/8652afb4ec51960f886c9c7c3ee96bd9a7c78f54/src/NServiceBus.Core/Routing/DistributionStrategy.cs#L25 the previous method comment. If you think we should align happy to change both